tests-llvm/ancient/LLVMTests.scala

```diff
@@ -0,0 +1 @@
+../jvm/src/test/scala/effekt/LLVMTests.scala
```

Comment:
What is this file?

Reply:
A symlink to the relevant testing code.

Reply:
Usually, you would run `sbt testOnly effekt.LLVMTests` to only run those tests.

Reply:
That is what we test in https://github.com/effekt-lang/effekt/blob/5d4058b8be40296016c8d635cf1157c041aef8fd/.github/workflows/ci.yml#L33; this symlink was a development relic. Removed in 5d4058b.